crates/types/src/storage/encode.rs

```diff
@@ -78,3 +78,8 @@ fn encode_serde_as_json<T: Serialize>(
 
     Ok(())
 }
+
+/// Utility method to encode a [`bilrost::Message`] type
+pub fn encode_bilrost<T: bilrost::Message>(value: &T) -> Bytes {
+    value.encode_fast().into_vec().into()
```

Comment:
Would it still make sense to use encode_fast? Would something like [encode_contiguous](https://docs.rs/bilrost/latest/bilrost/trait.Message.html#tymethod.encode_contiguous) be a better choice?

I'm also wonder if we should be using the  length-delimited variants of these functions, would that help or hurt us?

Reply:
`encode_contiguous` is an interesting option. But I am wondering why do you think the `length-delimited` variant would be useful ?

Reply:
length-delimited was prior to realizing that you the outer envelope is another bifrost message. Still curious to hear your thoughts on contiguous.

Reply:
I think `contiguous` is a good option. Since we need to convert the output to `vec` anyway after encoding. according to the docs `ReverseBuffer::into_vec` If this buffer is contiguous and has no spare capacity, the conversion will be performed without copying data. So this might be a nice improvement (the question is why it's not the default for `fast_encode()`).
Anyway, i changed the default net bilrost encoding now to use `encode_contiguous` it's part of this PR.